op-e2e/interop/supersystem_l2.go

```diff
@@ -215,6 +215,37 @@ func (s *interopE2ESystem) newGethForL2(id string, node string, l2Out *interopge
 	return l2Geth
 }
 
+func (s *interopE2ESystem) newProposerForL2(
+	id string, operatorKeys map[devkeys.ChainOperatorRole]ecdsa.PrivateKey) *l2os.ProposerService {
+	key := operatorKeys[devkeys.ProposerRole]
+	logger := s.logger.New("role", "proposer"+id)
+	proposerCLIConfig := &l2os.CLIConfig{
+		L1EthRpc:          s.l1.UserRPC().RPC(),
+		SupervisorRpc:     s.Supervisor().RPC(),
+		DGFAddress:        s.worldDeployment.L2s[id].DisputeGameFactoryProxy.Hex(),
+		ProposalInterval:  6 * time.Second,
+		DisputeGameType:   1, // Permissioned game type is the only one currently deployed
```

Comment:
Presumably https://github.com/ethereum-optimism/optimism/issues/11886 will fix this so that the op-proposer proposes super roots?

Reply:
doh! We don't even yet have the super dispute game type wired up.